core/tauri-utils/src/config.rs

```diff
@@ -1141,6 +1155,21 @@ pub struct SecurityConfig {
   /// Your application might be vulnerable to XSS attacks without this Tauri protection.
   #[serde(default, alias = "dangerous-disable-asset-csp-modification")]
   pub dangerous_disable_asset_csp_modification: DisabledCspModificationKind,
+  /// Allow external urls to send command to Tauri.
+  ///
+  /// By default, external urls do not have access to `window.__TAURI__`, which means they cannot
+  /// communicate with the commands defined in Rust. This prevents attacks where an externally
+  /// loaded malicious or compromised sites could start executing commands on the user's device.
+  ///
+  /// This configuration allows a set of external urls to have access to the Tauri commands.
+  /// Wildcards patterns are accepted and can be used to allow either all paths or
```

Comment:
AFAIK if you use an URL, this is not true anymore. But like I said I prefer the glob.

Reply:
It should still work, we use the same logic for the HTTP allowlist.

Reply:
I will need to test but I am pretty certain the Url parser rejects a single `*`?

Reply:
yes but you can do `https://*` (or `https://**`?) with effectively the same effect.

Reply:
I guess this is good for official TaurI, wont cut it for us since we also need to support http but I will maintain my fork.

Reply:
well, you can also do `http://...` of course. Sure, it's a bit annoying because it only accepts a single url (especially if we take into account the other convo about finer allowlist-like control)

Reply:
Yup, thus my recommendation to keep it as a free string

Reply:
Allowing `http` is super dangerous. And we could also allow an array of urls on the same scope.

Reply:
I mean sure but at the same time this feature is an escape hatch and its made abundantly clear with the name that its dangerous. Http is perfectly acceptable for a lot of use cases like local installations or forwarded via an SSH tunnel.